src/stories/button.stories.js

```diff
@@ -0,0 +1,71 @@
+import React from "react"
+import Button from "../components/button"
+import { action } from "@storybook/addon-actions"
+
+export default {
+  title: "Button",
+  component: Button,
+  args: {
+    buttonName: "Default",
+    label: "Storybook",
+  },
+  argTypes: {
+    buttonName: {
+      table: {
+        disable: true,
+      },
+    },
+  },
+}
+
+const Template = (args) => {
+  return (
+    <div className="mx-30 mb-40">
+      <div className="text-2xl mb-20">{args.buttonName}</div>
```

Comment:
we could extract the title and the container into some helper component that we use for displaying our stories. I found myself repeating this when migrating another component

Reply:
I tried something but it was not conclusive. I am not sure how to this

Reply:
let's migrate 2-3 components and a solution might present itself. I have something in mind that I could make a proposal for